integrations/GoogleAds/browser.js

```diff
@@ -48,44 +52,95 @@ class GoogleAds {
     logger.debug("===in init Google Ads===");
   }
 
-  identify(rudderElement) {
+  identify() {
     logger.debug("[GoogleAds] identify:: method not supported");
   }
 
   // https://developers.google.com/gtagjs/reference/event
   track(rudderElement) {
     logger.debug("in GoogleAdsAnalyticsManager track");
-    const conversionData = this.getConversionData(
-      this.clickEventConversions,
-      rudderElement.message.event
-    );
-    if (conversionData.conversionLabel) {
-      const { conversionLabel } = conversionData;
-      const { eventName } = conversionData;
-      const sendToValue = `${this.conversionId}/${conversionLabel}`;
-      const properties = {};
+    if (!this.dynamicRemarketing) {
+      const conversionData = this.getConversionData(
+        this.clickEventConversions,
+        rudderElement.message.event
+      );
+      if (conversionData.conversionLabel) {
+        const { conversionLabel } = conversionData;
+        const { eventName } = conversionData;
+        const sendToValue = `${this.conversionId}/${conversionLabel}`;
+        let properties = {};
+        if (rudderElement.message.properties) {
+          properties.value =
+            rudderElement.message.properties.revenue ||
+            rudderElement.message.properties.value;
+          properties.currency = rudderElement.message.properties.currency;
+          properties.transaction_id = rudderElement.message.properties.order_id;
+        }
+        properties.send_to = sendToValue;
+        properties = removeUndefinedAndNullValues(properties);
```

Comment:
Please add comments throughout the new code

Reply:
added comments